.github/workflows/go-getter.yml

```diff
@@ -12,8 +12,7 @@ jobs:
     strategy:
       matrix:
         go-version:
-          - 1.18
-          - 1.19
+          - 1.23
```

Comment:
could we maybe bump up to 1.25 already since that's the latest version available?